src/sentry_opsgenie/plugin.py

```diff
@@ -66,22 +66,27 @@ def get_form_initial(self, project=None):
             'alert_url': 'https://api.opsgenie.com/v1/json/alert',
         }
 
-    # TODO(dcramer): this is duplicated from sentry-webhooks
-    def get_group_data(self, group, event):
-        data = {
-            'id': str(group.id),
-            'checksum': group.checksum,
-            'project': group.project.slug,
-            'project_name': group.project.name,
-            'logger': group.logger,
-            'level': group.get_level_display(),
-            'culprit': group.culprit,
+    def build_payload(self, group, event):
+        payload = {
             'message': event.message,
-            'url': group.get_absolute_url(),
+            'alias': getattr(group, 'message_short', group.message).encode('utf-8'),
```

Comment:
We could also in theory use the gorup.id here, which would mean it relies purely on Sentry's grouping.